main.go

```diff
@@ -14,13 +14,14 @@ import (
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/filters"
 	"github.com/docker/docker/client"
+	"gopkg.in/matryer/try.v1"
 )
 
 var port = flag.Int("p", 8080, "Port to bind at")
 
 func main() {
 	flag.Parse()
-	log.Printf("Starting on port %d...", *port)
+	log.Println("Pinging Docker...")
```

Comment:
the logging changes are unrelated a bit but a good general improvement

Reply:
I agree, I think it would be even a good idea to log the actual prune operation as well, potentially with the retry number, just as some telemetry?

Reply:
Will add 👍